client/src/controllers/AutoSizeController.test.js

```diff
@@ -0,0 +1,48 @@
+import { Application } from '@hotwired/stimulus'
+import AutosizeController from "./AutoSizeController"
+
+const application = Application.start()
+application.register("autosize", AutosizeController)
```

Comment:
It's probably a good idea to use the identifier the same as used in other code. `w-auto-size`. While not strictly required for this test, it usually makes it easier to spot bugs and serves as a mini-documentation for other contributors.